docs/source/methods/energy_deposition.rst

```diff
@@ -0,0 +1,134 @@
+.. _methods_heating:
+
+=============================
+Heating and Energy Deposition
+=============================
+
+As particles traverse a problem, some portion of their energy is deposited at 
+collision sites. There are a variety of mechanisms that contribute to the
+energy deposition, including down-scattering from higher to lower energies,
+fission events, and reactions with positive Q-values. The information describing
```

Comment:
To be more precise, the heating really comes from charged particles (electrons, alphas, recoil nuclei) that undergo electromagnetic interactions with surrounding electrons and ions.